research/src/pages/component-spec-template.mdx

```diff
@@ -0,0 +1,213 @@
+> This document represents a component specification template. Its purpose
+> is to provide a point-in-time guide as to how a component should be
+> authored, in light of common anatomies, behaviors, and accessibility.
+> The primary audience is a developer authoring a given component.
+
+# {ComponentName} Component Specification
+
+## Overview
+
+*The name of the component, along with a high-level description.*
+
+### Background
+
+*Relevant historical or background information, related existing issues, etc.*
+
+### Use Cases
+
+*Primary use cases for this component.*
+
+### Non-goals
+
+*A list of use cases, features, or functionality which are **not** goals for the component.*
+  
+### Features
+
+*A list of the key features unique to this component.*
+
+### Risks and Challenges
+
+*Notable risks or challenges associated with implementing or shipping the component.*
+
+### Prior Art/Examples
+
+*Screenshots and/or links to existing, canonical, or exemplary implementations of the component.*
```

Comment:
Here, we should just be able to link to the research page.  It has ample examples of names, anatomy, concepts, and links to the implementations.  We plan to continue extending this page to show more and more information as well.

Reply:
Makes sense.

Reply:
I've updated the wording to indicate the research page should be linked and that if any additional examples are found while writing the spec, they should be submitted through a separate PR to the research page.